### ISSUE TITLE
Dev UI: email update

### ISSUE THREAD

**Issue by sitapriyamoorthi** [FredHutch/shiny-cromwell#187]
The email you get when your server starts says that contact scicomp is you have issues? Is this something we still want to pursue? Maybe point them to the github repo instead?

**Comment by sckott** [comment]
yeah i'd prefer github issues, but at least some folks wanted to do email. @dtenenba would have to change this if we did

**Comment by sitapriyamoorthi** [comment]
The dev app currently has these resources listed. I guess we should simply be consistent about whats in the email and whats on the repo. Just thinking of ways to streamline the process so relevant emails go to relevant folks!
![Screenshot 2024-11-13 at 10 15 49 AM](https://github.com/user-attachments/assets/eafee40a-3c24-49d0-bd90-384b8013d2f3)

**Comment by sckott** [comment]
You're right @dtenenba that not everyone will have a gh account, so let's keep an email option for sure.
